github.go

```diff
@@ -83,7 +89,29 @@ func checkOpenPRs(ctx *context.Context, client *github.Client, owner string, rep
 	}
 }
 
-func daysSincePRCreated(CreatedAt *time.Time) int {
+func initialDate(ctx *context.Context, client *github.Client, owner string, repo string, pullRequest *github.PullRequest) (*time.Time, error) {
+	if !runConfig.DaysSinceLastCommit {
+		return pullRequest.CreatedAt, nil
+	}
+	return lastCommitDate(ctx, client, owner, repo, pullRequest)
+}
+
+func lastCommitDate(ctx *context.Context, client *github.Client, owner string, repo string, pullRequest *github.PullRequest) (*time.Time, error) {
+	commitsList, _, err := client.PullRequests.ListCommits(*ctx, owner, repo, pullRequest.GetNumber(), nil)
```

Comment:
I'm not sure but maybe [GetUpdatedAt](https://godoc.org/github.com/google/go-github/github#PullRequest.GetUpdatedAt) would be the thing to use?

Reply:
GetUpdatedAt is related to everything in the PR, so if someone just edits the PR's description, this field gets updated.